app/assets/stylesheets/application.scss

```diff
@@ -175,6 +176,7 @@ a:hover {
   background:none;
   border:none;
   color: $unvisited-color;
+  font-family: $nexa-font;
```

Comment:
Properties should be ordered background, border, color, cursor, font, font-family, padding